lib/index.d.ts

```diff
@@ -0,0 +1,801 @@
+import type { Response } from "node-fetch"
+export { Response }
```

Comment:
Move every export to the end of the file please.

Reply:
What do you mean by every export? Do you want to use `export { A, B, C }` to export all the public symbols? I think this is old-fashioned, having an `export` prefixed right at the declaration site is better and is a generally accepted and more modern idiomatic way of ecamscript modules export.
I've reordered the private symbols to appear before the public ones in https://github.com/octet-stream/dinky/pull/7/commits/e55c40ea66ea80087afc8bc472c75c5a5ebdf98d

Reply:
This is the way how I do prefer to declare modules. Stick with the project's code style, please.
Also, I prefer to omit spaces inside of a curly braces.